enhancements/machine-config/machine-config-node.md

```diff
@@ -0,0 +1,340 @@
+processes.
+
+The data created by this MachineConfigNode would look like the following:
+
+```console
+$ oc get machineconfignodes
+NAME                          UPDATED   UPDATEPREPARING   UPDATEINPROGRESS   UPDATEPOSTACTION   UPDATECOMPLETING   RESUMING
+ip-10-0-12-194.ec2.internal   True      False             False              False              False              False
+ip-10-0-17-102.ec2.internal   True      False             False              False              False              False
+ip-10-0-2-232.ec2.internal    True      False             False              False              False              False
+ip-10-0-59-251.ec2.internal   True      False             False              False              False              False
+ip-10-0-59-56.ec2.internal    True      False             False              False              False              False
+ip-10-0-6-214.ec2.internal    True      False             False              False              False              False
+```
+
+as well as 
+
+```console
+$ oc get machineconfignodes -o wide
+NAME                          UPDATED   UPDATEPREPARING   UPDATEINPROGRESS   UPDATEPOSTACTION   UPDATECOMPLETING   RESUMING   COMPARINGMACHINECONFIGS   UPDATINGFILESANDOS   DRAININGNODE   CORDONINGNODE   REBOOTINGNODE   RELOADINGCRIO
+ip-10-0-1-37.ec2.internal     True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-33-243.ec2.internal   True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-37-208.ec2.internal   True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-39-225.ec2.internal   False     False             False              True               False              False      False                     False                False          False           True            False
+ip-10-0-51-162.ec2.internal   True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-59-150.ec2.internal   False     False             False              True               False              False      False                     False                False          False           True            False
+```
+
+Where each name represents a node in a given machineconfigpool. The statuses reported are created explicitly from MCO node annotations and MCO actions, no other operator actions are taken into account here. This allows us to get quite specific in what is occuring on the nodes.
```

Comment:
nit: I don't understand the "in a given machineconfigpool" part of the first sentence.  Can we drop that?  Or is there some pool filtering going on that needs more discussion?

Reply:
And to that, the machineconfigpool name will end up in the MCN spec, can we have it as a custom printer column so we can see which pool a node is in easily?

Reply:
+1 to machineconfigpool name

Reply:
There is no column for MCP in the 4.15 or 4.16 version. I can open a card to track this